packages/issuer/lib/VcIssuer.ts

```diff
@@ -72,9 +77,19 @@ export class VcIssuer {
     const { clientId, grants } = await this.retrieveGrantsFromClient(issuerState)
     await this.validateJWT(issueCredentialRequest, grants, clientId, jwtVerifyCallback)
     if (this.isMetadataSupportCredentialRequestFormat(issueCredentialRequest.format)) {
+      const cNonce = v4()
```

Comment:
We should probably start adding an optional opts arg with an optional nonce value. Use that in case it is present, otherwise generate it

Reply:
Fixed!